packages/trpc/server/routers/viewer/teams/create.handler.ts

```diff
@@ -45,6 +47,15 @@ export const createHandler = async ({ ctx, input }: CreateOptions) => {
     return duplicatedRequest;
   }
 
+  let parentId: number | null;
+  // If the user in session is part of an org. check permissions
+  if (ctx.user.organization?.id) {
+    if (!isOrganisationAdmin(ctx.user.id, ctx.user.organization.id)) {
+      throw new TRPCError({ code: "UNAUTHORIZED" });
+    }
+    parentId = ctx.user.organization.id;
+  }
```

Comment:
This file creates a team - we mayaswell do this in one place - teams.create and check if the user currently has a session
instead of deciding what to call on the FE